src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpCollectionExpressionRewriter.cs

```diff
@@ -61,6 +61,7 @@ internal static class CSharpCollectionExpressionRewriter
 #endif
 
         var initializer = getInitializer(expressionToReplace);
+        var endOfLine = EndOfLine(formattingOptions.NewLine);
```

Comment:
📝 This is not the correct end-of-line character for operations that are a subset of the document. It should instead copy the end-of-line character already present at the end of the current line.

Reply:
> This is not the correct end-of-line character for operations that are a subset of the document. It should instead copy the end-of-line character already present at the end of the current line.

That violates any sort of VS setting, or editorconfig setting that state what the preferred newline is.

Reply:
➡️ Resolved by using the existing EOL from the first line being changed by the refactoring.